sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/CountIf.scala

```diff
@@ -34,8 +34,11 @@ import org.apache.spark.sql.types.{AbstractDataType, BooleanType}
   """,
   group = "agg_funcs",
   since = "3.0.0")
-case class CountIf(child: Expression) extends RuntimeReplaceableAggregate
-  with ImplicitCastInputTypes with UnaryLike[Expression] {
+case class CountIf(child: Expression)
+    extends RuntimeReplaceable
+    with ReplaceableAggregateFunction
```

Comment:
I checked runtime replaceable aggregate functions. Seems like none requires `InheritAnalysisRules` for now.